frontend/ui/widget/configdialog.lua

```diff
@@ -486,8 +486,11 @@ function ConfigOption:init()
             end
 
             -- Icons (ex: columns, text align, with PDF)
-            if self.options[c].item_icons then
-                local items_count = #self.options[c].item_icons
+            local item_icons = self.options[c].item_icons_func and
+                               self.options[c].item_icons_func(self.config.configurable, self.config.document) or
```

Comment:
The 2nd argument is not used. Early idea abandonned, or ready for future ideas?
And even the 1st one, although passed, is not used in cre/koptions.lua.

Reply:
Yeah, that's to make the framework as versatile as the other `*_func` varian,t even if it isn't actually being used here.

(An earlier commit did try to get the value from the configurable, but it's fugly as hell, and less reliable).